### DISCUSSION TITLE
Is there some way to make a single page pdf with the title?

### DISCUSSION THREAD

**Discussion by ghost** [brechtm/rinohtype#358] [Q&A] [answered]
I have a single rst file that want to make a single page pdf with headline on the same page.

exemple:

~~~ rst
========
Headline
========

Lorem ipsum dolor sit amet, consectetur adipiscing elit,
sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
~~~

~~~ ini
[TEMPLATE_CONFIGURATION]
name = Checklist
template = article

table_of_contents=false

parts = 
    title
    contents

stylesheet = style.rts

[VARIABLES]
paper_size = A4
~~~

Is some way to make it work?

**Comment by brechtm** [comment, marked as answer]
This was not possible with the Article template up to now. That template was actually far too similar to the Book template, so I now adjusted to Article template to remove the separate title and front matter pages. See the `unnumbered_headings.*` files under https://github.com/brechtm/rinohtype/tree/master/tests_regression/rst for an example.

To make use of the new Article template before a new rinohtype release is made, install the in-development version:

    pip install https://github.com/brechtm/rinohtype/archive/refs/heads/master.zip
